sdk/src/main/scala/io/horizen/account/state/ForgerStakeMsgProcessorData.scala

```diff
@@ -297,3 +296,80 @@ object ForgerStakeDataSerializer extends SparkzSerializer[ForgerStakeData] {
     ForgerStakeData(forgerPublicKeys, ownerPublicKey, stakeAmount)
   }
 }
+
+case class StakeOfCmdInput(ownerAddress: Address) extends ABIEncodable[StaticStruct] {
+
+  override def asABIType(): StaticStruct = {
+
+    val listOfParams: util.List[Type[_]] = util.Arrays.asList(
+      new AbiAddress(ownerAddress.toString)
+    )
+    new StaticStruct(listOfParams)
+  }
+
+  override def toString: String = "%s(ownerAddress: %s)"
+    .format(
+      this.getClass.toString,
+      ownerAddress.toString)
+
+}
+
+object StakeOfCmdInputDecoder
+  extends ABIDecoder[StakeOfCmdInput]
+  with MsgProcessorInputDecoder[StakeOfCmdInput] {
+
+  override val getListOfABIParamTypes: util.List[TypeReference[Type[_]]] = {
+    org.web3j.abi.Utils.convert(util.Arrays.asList(
+      new TypeReference[AbiAddress]() {}
+    ))
+  }
+
+  override def createType(listOfParams: util.List[Type[_]]): StakeOfCmdInput = {
+    val address = new Address(listOfParams.get(0).asInstanceOf[AbiAddress].toString)
+    StakeOfCmdInput(address)
+  }
+}
+
+case class GetAllForgersStakesOfUserCmdInput(ownerAddress: Address) extends ABIEncodable[StaticStruct] {
```

Comment:
Do we really need two separate input/decoder pair objects for the new commands? After all we are handling the same data and we might think of using just one pair with a proper name.

Reply:
Done